tests/container_disk_test.go

```diff
@@ -207,13 +207,16 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				Expect(err).ToNot(HaveOccurred(), "expected alpine to login properly")
```

Comment:
I think there is a typo in the commit message `form`-->`from`.

Also, is it ok to add links to gihub PRs in our git repo commit message?
I mean, if tomorrow we decide to move this repo to gitlab (or the like) - these PR links will have no meaning.
in the PR description it's ok and good, but I think we should refrain from adding it the actual git commit description

Reply:
Fixed the typo.

Thanks for the explanation about the links in the commit message.
will remove those.